pkg/gcp/build/build.go

```diff
@@ -354,3 +358,28 @@ func mergeMaps(maps ...map[string]string) map[string]string {
 	}
 	return out
 }
+
+func ListJobs(project string, lastJobs int64) error {
+	ctx := context.Background()
+	opts := option.WithCredentialsFile("")
+
+	service, err := cloudbuild.NewService(ctx, opts)
+	if err != nil {
+		return errors.Wrap(err, "error fetching gcloud credentials... try running \"gcloud auth application-default login\"")
+	}
+
+	req, err := service.Projects.Builds.List(project).PageSize(lastJobs).Do()
+	if err != nil {
+		return errors.Wrap(err, "error listing the builds")
```

Comment:
Nit: let’s avoid the word “error” in error messages. Usually the user gets a `logrus.Fatalf` which already indicates that it’s an error.

Reply:
done, thanks for the review!